app/controllers/devise_registration_controller.rb

```diff
@@ -266,6 +302,26 @@ def edit_password; end
 
 protected
 
+  def create_jwt(payload)
+    jwt = Jwt.create!(
+      jwt_payload: payload,
+    )
+
+    session[:jwt_id] = jwt.id
+  end
+
+  def create_registration_state(payload, state, email, password, phone)
```

Comment:
Up to you, but maybe this method should be merged into `start` above as that's the only thing which uses it?  `start` is getting long now, but maybe the keyword arguments of `RegistrationState.create!` will be clearer than the positional arguments of `create_registration_state`?